<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="entityName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Secret">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>431</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Secure</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Send">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>434</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Listen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Host">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Load">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Send Img</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Send</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Secret</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>secret()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Listen</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>listen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Load</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Connect</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>send()</slot>
  <slot>listen()</slot>
  <slot>loadFile()</slot>
  <slot>connect()</slot>
  <slot>secret()</slot>
 </slots>
</ui>
